### ISSUE TITLE
Modifying Masking lengths size - BERT pretraining 

### ISSUE THREAD

**Issue by Swathygsb** [cdqa-suite/cdQA#206]
Is there any way to modify the length of random words that will be masked during BERT - pre training process?

**Comment by andrelmfarias** [comment]
@Swathygsb are you referring to the Masked Language Model (MLM) pretrain of BERT? The one performed by Google and explained in the official [paper](https://arxiv.org/pdf/1810.04805.pdf)?

**Comment by Swathygsb** [comment]
Yes. Do they provide any option to tweak that part?

**Comment by andrelmfarias** [comment]
I don't really know, as we use the pre-trained model we don't get to touch the MLM training of BERT. 

I suppose that, if you want to change it, you should train your own BERT Language model from scratch, which demands a lot of computing power (4 days on 16 TPU for BERT Base).

Unfortunately, that's not in the scope of our package... Therefore I am closing this issue
